src/main/kotlin/org/jitsi/nlj/dtls/DtlsStack.kt

```diff
@@ -220,10 +231,15 @@ class DtlsStack(
         val bufCopy = BufferPool.getBuffer(len).apply {
             System.arraycopy(data, off, this, 0, len)
         }
-        if (!incomingProtocolData.offer(ByteBuffer.wrap(bufCopy, 0, len))) {
-            logger.warn("DTLS stack queue full, dropping packet")
-            BufferPool.returnBuffer(bufCopy)
-            numPacketDropsQueueFull++
```

Comment:
Are we still dropping a packet here?

Reply:
No, and this was one of my questions as there's a change in behavior: previously the queue was fixed-size so offering could fail and we'd drop the packet.  We won't drop now, but the queue can grow.

Reply:
OK, I see now. I would prefer to avoid unbounded queues in case some unforeseen condition triggers a case where we keep adding. But if you keep it this way remove the `numPacketDropsQueueFull` counter.

Reply:
Yeah, I think I agree about the sized queue but thought I'd go the simple way (since enforcing a size will take some manual logic, unlike before and we could discuss.  I'll look at making that change.